src/Orleans/Serialization/SerializationContextBase.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+namespace Orleans.Serialization
+{
+    public static class SerializerContextExtensions
+    {
+        public static SerializationManager GetSerializationManager(this ISerializerContext context)
```

Comment:
Could we instead do the trick we do with GrainFactory and have the classes implement some `IInternalSerializerContext`? Perf would suffer less